packages/utils/src/phoneNumbers.ts

```diff
@@ -153,7 +153,9 @@ export function parsePhoneNumber(
         }
       : null
   } catch (error) {
-    console.debug(`phoneNumbers/parsePhoneNumber/Failed to parse phone number, error: ${error}`)
+    console.debug(
+      `phoneNumbers/parsePhoneNumber/Failed to parse phone number ${phoneNumberRaw}, error: ${error}`
```

Comment:
We intentionally excluded the phone number for privacy reasons, as the logs get uploaded to firebase